javascript/javascript3/week2/VisualPromise/main.js

```diff
@@ -0,0 +1,60 @@
+const redBox = document.querySelector("ul.marks li:nth-child(1)");
+const blueBox = document.querySelector("ul.marks li:nth-child(2)");
+const greenBox = document.querySelector("ul.marks li:nth-child(3)");
+
+const redTargetX = 20 - parseInt(redBox.style.left);
+const redTargetY = 300 - parseInt(redBox.style.top);
```

Comment:
Since these values are always used together, it would make sense to pair them, so you could write this instead:
```js
const redTarget = {
  x: 20 - parseInt(redBox.style.left),
  y: 300 - parseInt(redBox.style.top),
};
```

This gives two benefits, it shows that these two values relate to each other but also makes the call to the functions below easier to read:
`moveElement(redBox, redTarget)`